packages/react-reconciler/src/ReactFiberReconciler.new.js

```diff
@@ -707,6 +712,19 @@ function getCurrentFiberForDevTools() {
   return ReactCurrentFiberCurrent;
 }
 
+export function getIsStrictModeForDevtools() {
+  return isStrictMode;
+}
+
+export function setIsStrictModeForDevtools(newIsStrictMode: boolean) {
+  // We're in a test. Disable yielding during the double render
+  if (typeof Scheduler.unstable_yieldValue === 'function') {
+    Scheduler.unstable_setDisableYieldValue(newIsStrictMode);
+    setSuppressWarning(newIsStrictMode);
+  }
```

Comment:
I assume this check will work for Facebook tests too? (No reason to suspect it wouldn't, just thinking out loud.)